addons/l10n_eg_edi_eta_imp/models/account_move.py

```diff
@@ -0,0 +1,30 @@
+import json
+
+from odoo import api, models, fields
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_eg_long_id = fields.Char(string='ETA Long ID', compute='_compute_eta_response_data', store=True, copy=False)
```

Comment:
Why does this long_id need to be stored?  Because this entire calculation could be done in the l10n_eg_edi_eta module itself without storing.  (so it is fixed for everyone)

Reply:
isnt the creation of a field forbidden in stable? If so i can compute it in _compute_eta_response_data like the other fields (that are also stored for some reason)

Reply:
@bakboka There would not be a problem in doing a non-stored field (it is about the data model changing or not), but those should be in a different compute method than the stored ones.

Reply:
the issue if the field is not compute is that we have to open the json file and read it every time we access the view while we could just do it once at its creation